docs/dxp/latest/en/installation-and-upgrades/maintaining-a-liferay-dxp-installation/patching-liferay/accessing-dxp-patch-code.md

```diff
@@ -0,0 +1,111 @@
+# Accessing DXP Patch Code
+
+Liferay DXP Fix Pack and Hotfix source code is available in a private GitHub repository called `liferay-dxp`. There is a tagged branch for each Fix Pack and Hotfix. The source code facilitates identifying and understanding changes that may affect your project or your custom code. The figure below shows the GitHub Tags interface.
```

Comment:
For fix packs we are using 7.0.x/7.1.x/7.2.x/7.3.x and apply fix pack tags on them.
For hotfixes, each hotfix is on a different branch and they receive the hotfix tag at the branch head.

Reply:
Since a hotfix tag is at the hotfix branch head, is viewing the hotfix branch the same as viewing the hotfix branch + hotfix tag?

Reply:
... the code is identical?

Reply:
Yes, you can either fetch the hotfix tag or the hotfix branch, the result will be the same